oci-build-examples/oci-build-caching/README.md

```diff
@@ -0,0 +1,80 @@
+# OCI DevOps - Speed up builds with caching
+
+Most of the time the build time is consumed in downloading or preparing certain build dependancies like maven dependencies, node_modules, pip dependancies, etc... This document talks about on how to reduce time by caching those dependencies for the subsequent runs. 
+
+In this way, the first run would take full time to download and setup build dependancies. But the subsequent runs will be more efficient through the cache.
+
+For this, sample java maven-based application is used to demonstrate build caching.
+
+### Prerequisites/Assumptions
+* Assumed you are already familiar with OCI DevOps. Please refer [Documentation](https://www.oracle.com/devops/devops-service/)
+* To know how to run the this maven project in your local. Please refer [this](./SETUP-PROJECT.md)
+* Assumed that you are using docker build inside your `build_spec.yaml`
+
+### Overview of Changes
+For any project, you may tweek `build_spec.yaml` to enable build cache.
+
+#### Step 1:
+Docker BuildKit is installed to enable few advanced docker build commands for caching.
+```
+  - type: Command
+    name: "Docker BuildKit Setup"
+    timeoutInSeconds: 140
+    command: |
+      wget https://github.com/docker/buildx/releases/download/v0.8.2/buildx-v0.8.2.linux-amd64 -O docker-buildx
+      mkdir -p ~/.docker/cli-plugins
+      mv docker-buildx ~/.docker/cli-plugins/
+      chmod +x ~/.docker/cli-plugins/docker-buildx
+      docker buildx install
+```
+#### Step 2
+`Build Cache Restore` stage is used to download the pre-uploaded cache from OCI Object Storage.
+
```

Comment:
As we are using object storage ,its also key to add if any policies to be mentioned for object read.

Reply:
Added the info in the 'Prerequisites' section.